Tests/CurrencyTests/CurrencyValue+StringRepresentationTests.swift

```diff
@@ -149,15 +150,20 @@ extension CurrencyValueStringRepresentationTests {
   }
 
   func test_stringInterpolation_forLocale_usesDefaultLocale() {
-    XCTAssertEqual("\(localize: USD(4321.389))", "$4,321.39")
+    XCTAssertEqual("\(localize: USD(4321.389), for: Locale(identifier: "en_US"))", "$4,321.39")
   }
 }
 
 // MARK: LocalizedString
 
 extension CurrencyValueStringRepresentationTests {
   func test_localizedString_forLocale_usesDefaultLocale() {
-    XCTAssertEqual(USD(4321.389).localizedString(), "$4,321.39")
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .currency
+    formatter.locale = .current
+    formatter.currencyCode = USD.alphabeticCode
+    let expectedOutput = formatter.string(for: Decimal(4321.389))
```

Comment:
All this needs to be duplicated for `test_stringInterpolation_forLocale_usesDefaultLocale`